MathlibTest/ModuleCasing.lean

```diff
@@ -0,0 +1,33 @@
+import Mathlib.Tactic.Linter.TextBased
+
+/-!
+Unit tests for the module name case check in the text-based linters.
+
```

Comment:
```suggestion
```
I probably prefer this, but I'm happy if you want to merge with the extra line break as well!